---
title: LOCALDB_ERROR_INTERNAL_ERROR |Microsoft Docs
ms.custom: ''
ms.date: 06/13/2017
ms.prod: sql-server-2014
ms.reviewer: ''
ms.suite: ''
ms.technology: supportability
ms.tgt_pltfrm: ''
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
caps.latest.revision: 7
author: stevestein
ms.author: sstein
manager: craigg
ms.openlocfilehash: cbf3cbefc6844329127647e1024addc4c3d8ea9e
ms.sourcegitcommit: f8ce92a2f935616339965d140e00298b1f8355d7
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 07/03/2018
ms.locfileid: "37410072"
---
# <a name="localdberrorinternalerror"></a>LOCALDB_ERROR_INTERNAL_ERROR
    
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|264|  
|イベント ソース|SQL Server Local Database Runtime 12.0|  
|コンポーネント|Local Database Runtime API|  
|メッセージ テキスト|ローカル データベース インスタンスの API メソッドの呼び出しで予期しないエラーが発生しました。 エラーの詳細については、Windows アプリケーション ログを参照してください。|  
  
## <a name="explanation"></a>説明  
 予期しないエラーが発生しました。  
  
## <a name="user-action"></a>ユーザーの操作  
 詳細をイベント ログで確認してください。  
  
  
